linux_os/guide/system/software/integrity/software-integrity/aide/aide_check_audit_tools/rule.yml

```diff
@@ -33,14 +33,16 @@ severity: medium
 identifiers:
     cce@sle12: CCE-83204-8
     cce@sle15: CCE-85610-4
+    cce@rhel8: CCE-85964-5
 
 references:
     disa: CCI-001496
-    nist@sle12: AU-9(3),AU-9(3).1
+    nist: AU-9(3),AU-9(3).1
     srg: SRG-OS-000278-GPOS-00108
     stigid@sle12: SLES-12-010540
     stigid@sle15: SLES-15-030630
     stigid@ubuntu2004: UBTU-20-010205
+    stigid@rhel8: RHEL-08-030650
```

Comment:
tests are indicating that references are not sorted:
`linux_os/guide/system/software/integrity/software-integrity/aide/aide_check_audit_tools/rule.yml has one or more unsorted references`
https://github.com/ComplianceAsCode/content/pull/7283/checks?check_run_id=3147487014